zabbix_cli/cli.py

```diff
@@ -38,12 +41,13 @@
 import textwrap
 import time
 
-import ipaddr
-
 import zabbix_cli
+import zabbix_cli.version
```

Comment:
The version module has been removed. I guess the import got erroneously merged back in?

Reply:
zabbix-cli now crashes with: `ModuleNotFoundError: No module named 'zabbix_cli.version'`

Reply:
Probably a merge-issue. Fixed!